lib/platform-stats.js

```diff
@@ -199,3 +316,119 @@ export const updateTopMeasurementParticipants = async (createPgClient) => {
     Sentry.captureException(err)
   }
 }
+
+/**
+ * @param {import('pg').Client} pgClient
+ */
+export const aggregateAndCleanupRecentData = async (pgClient) => {
+  try {
+    await pgClient.query('BEGIN')
+
+    // The ON CONFLICT theoretically should not happen, because it's two days old data,
+    // which should not be updated anymore. But it's a good practice to have it.
+    await pgClient.query(`
+      WITH rsdn_summary AS (
+        SELECT
+          day,
+          SUM(accepted_measurement_count) AS accepted_measurement_count,
+          SUM(total_measurement_count) AS total_measurement_count,
+          COUNT(DISTINCT station_id) AS distinct_active_station_count,
+          COUNT(DISTINCT participant_id) AS distinct_participant_address_count
+        FROM
+          recent_station_details
+        WHERE
+          day <= CURRENT_DATE - INTERVAL '2 days'
+        GROUP BY
+          day
+      ),
+      rps_summary AS (
```

Comment:
What does `rps` stand for`?

Reply:
`recent_participant_subnets` table

Reply:
ok, what do you think about just writing that out?

Reply:
Addressed in https://github.com/filecoin-station/spark-evaluate/pull/311/commits/e3845c9bd36cfddb719459fe72c3f2389b94ecda